src/DetailPage.js

```diff
@@ -0,0 +1,35 @@
+import React, { Component } from 'react'
+import request from 'superagent';
+
+export default class DetailPage extends Component {
+    state = {
+        chosenCharacter: {}
+    }
+
+    componentDidMount = async () => {
+        const data = await request.get(`https://dizknee-express.herokuapp.com/characters/`);
```

Comment:
You should be able to hit your characters/:name endpoint using the character name from the params. Then you won't need to find the character in the results on line 13. :)